_docs/backenders/strategy-2021/roles/quality.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: Quality
+parent: Roles
+grand_parent: Strategy 2021
+---
+
+# Quality role definition
+
+Delivering high-quality work is a trademark of Vizzuality. Ensuring clients and
+users are happy using the products we build is something we need to pay 
+attention in everything we do daily. By delivering high-quality work, on time 
+and efficiently, we keep clients happy and we can be proud of the work we do.
+
+This role is the point of contact between the Backend and on-going Quality 
+initiatives at Vizzuality.
+
+**Current responsible**: Henrique Pacheco
```

Comment:
Minor one: I have volunteered to share with you / help with the responsabilities below (you were off that meeting I think), not sure if you want to reflect that, up to you 😃

Reply:
Oh, that's great, I wasn't aware of that :smile: I'm more than happy to share the responsibility :sweat_smile: updated current responsible :rocket: